recipes/ksonnet/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "ksonnet" %}
+{% set version = "0.13.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ksonnet/ksonnet/releases/download/v{{ version }}/ks_{{ version }}_darwin_amd64.tar.gz  # [osx]
+  sha256: 5e4bf4ef3eec3c47082c95ded07fd32b70c214134626bca8d7a6a424ea7ffeb8  # [osx]
+
+  url: https://github.com/ksonnet/ksonnet/releases/download/v{{ version }}/ks_{{ version }}_linux_amd64.tar.gz  # [linux]
+  sha256: fe76e414e5cba3f093e34e1b433f79cd53915e3153b68cff5ba3273cafd8a2a0  # [linux]
+
+  url: https://github.com/ksonnet/ksonnet/releases/download/v{{ version }}/ks_{{ version }}_windows_amd64.zip  # [win]
+  sha256: ca76063a2819ade0c4e2749c59fb92cfd7210d1c1196e7b4c20b0f4b4bd2bebc  # [win]
+
+build:
+  number: 0
+
+test:
+  commands:
+    - ks -h  # [unix]
+    - ks.exe -h  # [win]
+
+about:
+  home: https://ksonnet.io/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'A CLI-supported framework for extensible Kubernetes configurations'
+
+  description: |
+    ksonnet is a framework for writing, sharing, and deploying Kubernetes application manifests. With its CLI, you can generate a complete application from scratch in only a few commands, or manage a complex system at scale.
+    Specifically, ksonnet allows you to:
+    * Reuse common manifest patterns (within your app or from external libraries)
+    * Customize manifests directly with powerful object concatenation syntax
+    * Deploy app manifests to multiple environments
+    * Diff across environments to compare two running versions of your app
+    Track the entire state of your app configuration in version controllable files
+    All of this results in a more iterative process for developing manifests, one that can be supplemented by continuous integration (CI).
+  doc_url: https://ksonnet.io/docs/
+  dev_url: https://ksonnet.io/docs/
```

I guess this should be:
```suggestion
  dev_url: https://github.com/ksonnet/ksonnet
```
Also is there a reason for repackaging binaries rather than building from source?

---

@chrisburr Thanks for the feedback!

> Also is there a reason for repackaging binaries rather than building from source?
1. We assume, it's an easier way to get things done.
2. Further, ksonnet is currently abandoned and we would rather abstain from investing more work in that task, than necessary.  
3. We find a number of packages that rather repackage a binary, than build from source.

---

It's mostly to make it easier on the packaging side.

Given that these are Go binaries, I don't think it is an issue.